Unable to build on EndeavourOS (and most likely Arch Linux)

It seems impossible to build under EndeavourOS with the current Makefile.

I am getting an error an error like :
 ```
/usr/bin/ld: /tmp/ccV4wprr.o:(.bss+0x0): multiple definition of `i'; /tmp/ccBdLCoq.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
make: *** [<builtin>: conky-colors] Error 1
```

As pointed out [here](https://www.linuxquestions.org/questions/programming-9/multiple-definition-errors-when-linking-on-arch-using-gcc-10-1-0-a-4175675444/), it seems the default compiler flag is now `-fno-common` which does not allow such duplicates. 

The workaround, in my case is to add the flag `-fcommon` to the `$CFLAGS` variable in the `Makefile`.

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 PREFIX=/usr/local
 VPATH=src/
-CFLAGS:=$(CFLAGS) -DDESTDIR=\"$(DESTDIR)$(PREFIX)\" -Wall -std=c99
+CFLAGS:=$(CFLAGS) -fcommon -DDESTDIR=\"$(DESTDIR)$(PREFIX)\" -Wall -std=c99
 CWD:=$(shell pwd)
 
 all: conky-colors

